scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -827,7 +827,7 @@ class Router(formatOps: FormatOps) {
         val defnSite = !tupleSite && anyDefnSite
 
         val indent =
-          if (anyDefnSite && !leftOwner.isInstanceOf[Type.Apply])
```

Comment:
Wasn't this because `Type.Apply` is actually only present in definitions?

It also seems that we will get a lot of changes if people are using default configuration.

Reply:
fist, this specific line doesn't change anything, it just removes the no longer necessary check because Type.Apply is not a defnSite anymore.

also, `Type.Apply` can be present in both definitions and not, but _itself_ it is *not* a definition site.

```
  // used within a call site
  val a = foo(bar: Map[String, String])

  // used within a defn site but on the rhs, hence not a defn
  type a = Map[String, String]
```

Reply:
Cool! Makes sense.